CONTRIBUTING.md

```diff
@@ -8,13 +8,60 @@ We use Node in this project. To manage the language runtime we recommend using [
 
 With `nvm` use `nvm use` to set the correct node version in your machine.
 
+To setup the Android environment install Android studio. Once installed, use the following commands to setup the android sdk
+
+```
+echo "export ANDROID_HOME=\"$HOME/Library/Android/sdk\"" >> ~/.bash_profile
+echo "export ANDROID_SDK_ROOT=\"$HOME/Library/Android/sdk\"" >> ~/.bash_profile
```

Comment:
we use `ANDROID_SDK_ROOT` because of some react native dependency if I remember correctly. We don' t really need it here but I guess is better to advocate it as it will make the setup work for react native too. That being said we should update the commands bellow to use either all `ANDROID_HOME` or `ANDROID_SDK_ROOT`

Reply:
Android studio sets `$ANDROID_HOME` by default. Travis and RN use `$ANDROID_SDK_ROOT` . I am setting both but will update the commands to use one